modules/screenshots/module.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+namespace Elementor\Modules\Screenshots;
+
+use Elementor\Core\Files\CSS\Post_Preview;
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\User;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Module extends BaseModule {
+
+	public function get_name() {
+		return 'screenshots';
+	}
+
+	public function screenshot_proxy() {
+		if ( ! wp_verify_nonce( $_GET['nonce'], 'screenshot_proxy' ) ) {
+			echo '';
+		}
+
+		if ( ! empty( $_GET['href'] ) ) {
+			$response = wp_remote_get( utf8_decode( $_GET['href'] ) );
+			$body = wp_remote_retrieve_body( $response );
+			if ( $body ) {
+				echo $body;
+			}
+		}
+	}
+
+	public function ajax_save( $data ) {
+		if ( empty( $data['screenshot'] ) ) {
+			return false;
+		}
+
+		$post_id = $data['post_id'];
+
+		$file_content = substr( $data['screenshot'], strlen( 'data:image/png;base64,' ) );
+		$file_name = 'Elementor Post Screenshot ' . $post_id . '.png';
+		$over_write_file_name_callback = function () use ( $file_name ) {
+			return $file_name;
+		};
+
+		add_filter( 'wp_unique_filename', $over_write_file_name_callback );
+
+		$upload = wp_upload_bits(
```

Comment:
Try move all screenshots to a new folder `/wp-content/uploads/elementor/screenshots`. (create if not exist with `wp_mkdir_p`)